### PR TITLE
Netlify CMS Setup

### PR DESCRIPTION
**Pull request by shailen-naidoo**

### CODE REVIEW

File: admin/config.yml [hunk 1]
```diff
@@ -1,19 +1,43 @@
 backend:
   name: github
   repo: fedsa/fedsa-website
-  # branch: master
+  branch: netlify-cms-setup
```
Comment (schalkventer) [inline admin/config.yml:4]: Should you not change this back?

File: pages/index.js [hunk 2]
```diff
@@ -1,4 +1,13 @@
 const html = require('html-template-tag')
+const homepageData = require('../public/data/homepage.json')
+const md = require('markdown-it')()
+
+const Blurb = ({ title, body }) => html`
+  <div class="Blurb">
+    <h2 class="Blurb-title">${title}</h2>
+    $${md.render(body)}
```
Comment (schalkventer) [inline pages/index.js:8]: Does this do escaping as well?

File: pages/index.js [hunk 3]
```diff
@@ -1,4 +1,13 @@
 const html = require('html-template-tag')
+const homepageData = require('../public/data/homepage.json')
+const md = require('markdown-it')()
+
+const Blurb = ({ title, body }) => html`
+  <div class="Blurb">
+    <h2 class="Blurb-title">${title}</h2>
```
Comment (schalkventer) [inline pages/index.js:7]: Also make sure that this is escaped properly.

File: admin/config.yml (hunk 1 above)
Comment (shailen-naidoo) [reply admin/config.yml:4]: Yeah will do :smile:

File: pages/index.js (hunk 2 above)
Comment (shailen-naidoo) [reply pages/index.js:8]: Yebo yes

`$${whater}` adding the extra `$` sign before the interpolated variable unescapes the HTML

File: pages/index.js (hunk 3 above)
Comment (shailen-naidoo) [reply pages/index.js:7]: Thanks, will make sure :)

**Pull request merged**